src/containers/Programs/index.tsx

```diff
@@ -176,7 +178,7 @@ const Programs = ({
 
   useEffect(() => {
     if (information) setStateToRender(information);
-  }, [information]);
+  }, [information, stateToRender]);
```

Comment:
Why are we using the state to render in dependency?

Reply:
yes as this updates the list